vyper/utils.py

```diff
@@ -148,7 +148,7 @@ def in_bounds(cls, type_str, value):
     'pass', 'def', 'push', 'dup', 'swap', 'send', 'call',
     'selfdestruct', 'assert', 'stop', 'throw',
     'raise', 'init', '_init_', '___init___', '____init____',
-    'true', 'false', 'self', 'this', 'continue',
+    'true', 'false', 'self', 'this', 'continue', 'none',
```

Comment:
Should clear become a reserved keyword?

Reply:
Yup

Reply:
Just pushed a PR that added `clear` as a reserved word and added a entry to changelog.